release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -2399,3 +2408,66 @@ Issued: 2020-08-24
 An update for `openshift-enterprise-hyperkube-container` is now available for
 {product-title} 4.5. Details of the update are documented in the
 link:https://access.redhat.com/errata/RHSA-2020:3520[RHSA-2020:3520] advisory.
+
+[[ocp-4-5-8]]
+=== RHBA-2020:3510 - {product-title} 4.5.8 bug fix update
+
+[[ocp-4-5-8-features]]
+==== Features
+
+[[ocp-4-5-8-added-credentialsmode-parameter-for-aws-install-workaround]]
+===== Added credentialsMode parameter to bypass faulty permissions check during AWS installation
```

Comment:
I'd like to clarify a little here that this is AWS's bug, not ours. Maybe "to bypass AWS policy simulator shortcomings"?

Reply:
Fair enough! I'll tweak it :)